contributor_resources.rst

```diff
@@ -54,4 +54,4 @@ and the step can vary from one project to another. We recommend that you check o
 contributing guidelines of the project that you’re interested in, and pre-install the
 requirements before coming to the sprint. If you're stuck in any way, let us know so we
 can help. We have also requested each of the projects that signed up to have their
-contributing guidelines up to date by **April 26th**.
+contributing guidelines up to date by **April 3rd**.
```

Comment:
I would also propose putting my step by step guide for 1st pull request here: https://slides.com/cheukting_ho/1st_pr

Reply:
Awesome @Cheukting will add this!